hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/InodeTree.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.fs.UnsupportedFileSystemException;
+import org.apache.hadoop.fs.impl.FunctionsRaisingIOE.FunctionRaisingIOE;
```

Comment:
It looks like @steveloughran deprecated this class later after his comment. Don't know what prompted the refactoring, but
`import org.apache.hadoop.fs.impl.FunctionsRaisingIOE.FunctionRaisingIOE`
should be now
`org.apache.hadoop.util.functional.FunctionRaisingIOE`

Reply:
replaced `org.apache.hadoop.fs.impl.FunctionsRaisingIOE.FunctionRaisingIOE` with `org.apache.hadoop.util.functional.FunctionRaisingIOE`